inst/JOSS/paper.Rmd

```diff
@@ -187,18 +193,33 @@ ha_record <- bcdc_get_record("7bc6018f-bb4f-4e5d-845e-c529e3d1ac3b")
 ## Have a quick look at the geospatial columns to help with filter or select
 bcdc_describe_feature(ha_record)
```

Comment:
We should make sure to merge in master immediately prior to final submission to make sure that the paper reflects the most current state.